doc/syntax/Makefile

```diff
@@ -1,7 +1,11 @@
-syntax.pdf: syntax.tex
-	latexmk -pdf -halt-on-error -shell-escape -pvc syntax.tex
+syntax.pdf: syntax_en.pdf syntax_fr.pdf
+	cpdf $^ -o $@
```

Comment:
Ok hence the new `cpdf` dependency.

Reply:
To be honest joining the two cheat-sheets in a single pdf file could probably be done directly using some LaTeX / Postscript tricks… but I had enough with LaTeX already at this point ;)

Reply:
cpdf is licensed for [non-commercial use](https://github.com/johnwhitington/cpdf-source?tab=License-1-ov-file) only (non-OSS license), while Catala is licensed under Apache. Are they compatible?

Reply:
This should be ok, we don't ship it or include it in any way, just use the binary during builds of the tutorial, which is clearly non-commercial.
The use here is really just joining 2 pdf files though so there may be a simpler/better choice ?

Reply:
Like `pdfunite` from poppler-utils maybe